test/lib/usd/translators/testUsdExportUVSets.py

```diff
@@ -97,38 +105,36 @@ def setUpClass(cls):
         uvSetName = 'AllFacesSharedSet'
         (uArray, vArray) = meshFn.getUVs(uvSetName)
         (numUVShells, shellIndices) = meshFn.getUvShellsIds(uvSetName)
-        # These values are incorrect, in that they are not what's stored in the
-        # Maya scene, and not what we expect to export. We also use raw asserts
-        # here since we don't have an instance of unittest.TestCase yet.
-        assert(len(uArray) == 24)
-        assert(numUVShells == 6)
+
+        # The size of the UV array and shells might be incorrect,
+        # in that they are not what's stored in the Maya scene,
+        # and not what we expect to export. We fix this below.
 
         # Fix up the "all shared" UV set.
-        meshFn.clearUVs(uvSetName)
-        meshFn.setUVs(
-            [0.0, 1.0, 1.0, 0.0],
-            [0.0, 0.0, 1.0, 1.0],
-            uvSetName)
-        meshFn.assignUVs([4, 4, 4, 4, 4, 4], [0, 1, 2, 3] * 6, uvSetName)
-        (numUVShells, shellIndices) = meshFn.getUvShellsIds(uvSetName)
-        assert(numUVShells == 1)
+        if len(uArray) != 4 or numUVShells != 1:
+            meshFn.clearUVs(uvSetName)
+            meshFn.setUVs(
+                [0.0, 1.0, 1.0, 0.0],
+                [0.0, 0.0, 1.0, 1.0],
+                uvSetName)
+            meshFn.assignUVs([4, 4, 4, 4, 4, 4], [0, 1, 2, 3] * 6, uvSetName)
+            (numUVShells, shellIndices) = meshFn.getUvShellsIds(uvSetName)
+        self.assertEquals(numUVShells, 1)
```

Comment:
assertEquals is [deprecated](https://docs.python.org/3/library/unittest.html#deprecated-aliases). Can you use assertEqual instead?